src/main/java/io/excaliburfrc/robot/Constants.java

```diff
@@ -30,12 +30,23 @@
   public static class ClimberConstants {
     public static final int FORWARD_CHANNEL = 0;
     public static final int REVERSE_CHANNEL = 1;
+
+    public static final double MAX_HEIGHT = 10.0; // Rotations
+
     public static final int LEFT_MOTOR_ID = 51;
     public static final int RIGHT_MOTOR_ID = 52;
 
     public static final double kP = 0;
     public static final double kI = 0;
     public static final double kD = 0;
+    public static final double MG = 1000;
+    public static final double ANGLE = 45;
+    public static final double kS = 0;
+    public static final double kV = 0;
+    public static final double kA = 0;
+    public static final double kMaxV = 0.5;
+    public static final double kMaxA = 1;
```

Comment:
can you name these in SCREAMING_SNAKE_CASE, without the Hungarian-notation `k` prefix, and with 'veloctiy' instead of 'v' (same for acceleration)?